app/views/diabetes-content/health-problems-caused-by-diabetes-type-2.html

```diff
@@ -9,69 +9,89 @@
 {% markdown %}
 #Health problems caused by diabetes type 2#
 You need to watch your health and have regular check-ups if you have diabetes because high blood sugar can cause other problems. 
-
-It can damage your blood vessels, nerves and organs (eg heart and kidneys) and lead to:
+and lead to:
 
 - [heart disease](http://www.nhs.uk/conditions/coronary-heart-disease/pages/introduction.aspx) and [stroke](http://www.nhs.uk/conditions/coronary-heart-disease/pages/introduction.aspx)
-- pain, tingling and loss of feeling (nerve damage)
-- vision loss and blindness
+- loss of feeling and pain (nerve damage) - causing problems with sex
 - foot problems - sores and infections
-- loss of sensation and problems during sex
+- vision loss and blindness
 - miscarriage and stillbirth
 
-Some complications can be avoided. It’s important you know the signs and what steps you should take to prevent them.
+Controlling your blood sugar level and having regular diabetes check-ups is the best way to lower your risk of complications. 
+
+##Getting your heart checked##
+
+You should have your blood fats (cholesterol) and blood pressure checked at least once a year. This is part of your annual diabetes review. Diabetes increases your risk of developing heart disease and stroke so it’s important high blood pressure and high cholesterol is spotted and treated early. 
+
+[Book your annual diabetes review](http://nhsalpha.herokuapp.com/)
```

Comment:
The link here should be `http://localhost:3000/book-an-appointment/diabetes-annual-review/find-new-appointment` rather than `http://nhsalpha.herokuapp.com/`